certification/internal/engine/podman.go

```diff
@@ -4,66 +4,39 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	"io"
+	"os"
 	"os/exec"
+	"regexp"
 	"strings"
 
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/cli"
 	log "github.com/sirupsen/logrus"
 )
 
+const (
+	syetemdDir = "/etc/systemd/system"
```

Comment:
I think this is the `var` you were asking about, once my PR lands, you can place this in `defaults.go`

Reply:
Awesome! Thanks for making our life easier